llama-index-core/llama_index/core/response_synthesizers/context_only.py

```diff
@@ -0,0 +1,30 @@
+from typing import Any, Sequence
+
+from llama_index.core.prompts.mixin import PromptDictType
+from llama_index.core.response_synthesizers.base import BaseSynthesizer
+from llama_index.core.types import RESPONSE_TEXT_TYPE
+
+
+class ContextOnly(BaseSynthesizer):
+    def _get_prompts(self) -> PromptDictType:
+        """Get prompts."""
+        return {}
+
+    def _update_prompts(self, prompts: PromptDictType) -> None:
+        """Update prompts."""
+
+    def get_response(
+        self,
+        query_str: str,
+        text_chunks: Sequence[str],
+        **response_kwargs: Any,
+    ) -> RESPONSE_TEXT_TYPE:
+        return "\n\n".join(text_chunks)
```

Comment:
technically there is a prompt template here, this joiner could maybe be customizable?